google-cloud-tools-plugin/src/com/google/gct/idea/appengine/util/CloudSdkUtil.java

```diff
@@ -83,26 +83,38 @@ public static boolean containsCloudSdkExecutable(@NotNull String path) {
    * @param path @link String} to Cloud SDK binary on local file system.
    * @return a boolean indicating if the file was found.
    */
-  public static boolean isCloudSdkExecutable(@NotNull String path) {
-    VirtualFile vfile = LocalFileSystem.getInstance().findFileByPath(path);
-    return vfile != null && vfile.exists();
+  public static boolean isCloudSdkExecutable(String path) {
+    if (path != null) {
+      VirtualFile vfile = LocalFileSystem.getInstance().findFileByPath(path);
+      return vfile != null && vfile.exists();
+    }
+
+    return false;
   }
 
   /**
    * Converts from a parent directory path to the Cloud SDK executable path
    *
    */
-  public static String toExecutablePath(@NotNull String sdkDirectoryPath) {
-    File executablePath = new File(sdkDirectoryPath, getSystemCommand());
-    return executablePath.getAbsolutePath();
+  public static String toExecutablePath(String sdkDirectoryPath) {
```

Comment:
Because those utilities are being called to store values in the UI based on stored configuration (which can be null), so I decided to make the utilities null safe rather than null checks everywhere.